tests/integration/datadictwithobjid/conftest.py

```diff
@@ -0,0 +1,280 @@
+import os
+import json
+from multiprocessing import Process
+
+from indexd import default_settings, get_app as get_indexd_app
+from indexclient.client import IndexClient
+import pytest
+import requests
+import requests_mock
+from mock import patch
+from flask.testing import make_test_environ_builder
+from fence.jwt.token import generate_signed_access_token
+from psqlgraph import PsqlGraphDriver
+from gdcdatamodel.models import Edge, Node
+from userdatamodel import models as usermd
+from userdatamodel import Base as usermd_base
+from userdatamodel.driver import SQLAlchemyDriver
+from cdispyutils.hmac4 import get_auth
+from dictionaryutils import DataDictionary, dictionary
+from datamodelutils import models, validators
+
+import sheepdog
+import utils
+from sheepdog.auth import roles
+from sheepdog.test_settings import (
+    PSQL_USER_DB_CONNECTION,
+    Fernet,
+    HMAC_ENCRYPTION_KEY,
+    JWT_KEYPAIR_FILES,
+    SIGNPOST,
+)
+from tests.integration.datadictwithobjid.api import app as _app, app_init, indexd_init
+from tests.integration.datadictwithobjid.submission.test_endpoints import put_cgci_blgsp
+
+try:
```

Comment:
according to http://python-future.org/compatible_idioms.html#reload you can just do `from imp import reload`

Reply:
for python 2 and 3